src/liballoc/str.rs

```diff
@@ -1734,7 +1734,7 @@ impl str {
     /// A more complex pattern, using a closure:
     ///
     /// ```
-    /// assert_eq!("1fooX".trim_left_matches(|c| c == '1' || c == 'X'), "fooX");
+    /// assert_eq!("1fooX".trim_right_matches(|c| c == '1' || c == 'X'), "fooX");
```

Comment:
When trimming from the right, the result would no longer be `"fooX"`. I think it would be `"1foo"`. https://play.rust-lang.org/?gist=433e02bea040f9f2364621f3cf0830a4

Reply:
Yep I see it, nice response time tho.